release/README.md

```diff
@@ -3,48 +3,59 @@
 The releases will follow semantic versioning and be handled with git tags.
 https://semver.org/
 
-## Tracked releases
+## Major and minor releases
+
+1. To release a major or minor version create a release branch `release-X.Y` from the last minor release tag.
 
-When ready to cut a new major or minor release create a release branch
-`release-x.y` from the last minor release tag. For example:
 ```bash
-git checkout v0.2.0
-git checkout -b release-0.3
-git push -u origin release-0.3
+git checkout vA.B.0
+git checkout -b release-X.Y
+git push -u origin release-X.Y
 ```
-The rest of the workflow will be handled by a pipeline see
-[Trigger a release](#trigger-a-release).
 
-## Trigger a release
+2. Checkout the branch to cut the release from.
+
+```bash
+git checkout master
+git pull
+git checkout -b branch_name
+```
+
+3. Run the release script.
+
+```bash
+./release/release.sh vX.Y.Z
```

Comment:
```suggestion
./release/release.sh vX.Y.0
```

Since this is a major/minor release

Reply:
Haha funny, this is exactly what I did at first but I changed it so that it would fit with the patch releases as well (it points to step 3 in the major/minor release flow). :smile:

Should I change it back or do you think it's reasonable to keep it like this because of that?